types/events_test.go

```diff
@@ -7,6 +7,20 @@ import (
 	"github.com/stretchr/testify/assert"
 )
 
+// Verify that the event data types satisfy their shared interface.
+var (
+	_ TMEventData = EventDataCompleteProposal{}
```

Comment:
I see `EventDataBlockSyncStatus` and `EventDataStateSyncStatus` are not used here.
Rightly so, as they are not defined in this branch.
Would it make sense to backport to this feature branch #6755 and/or #6700, which defined then?

Reply:
maybe we can add TODO here to avoid miss when backport these feature branches later

Reply:
Please add the TODO, I think it's a good idea.
My question was rather, whether THIS is the right moment to bring on #6755 (and any other it depends on), and #6700.

Reply:
I could help to backport these 2 PRs later, not sure if we give them a priority.

Reply:
Fair enough. If you could help with it (backporting them in this branch), it would be awesome